docs/getting_started/requirements.md

```diff
@@ -103,12 +108,11 @@ These setups have been undergone some additional manual testing and is known to
 
 -   Mozilla® Firefox® most recent stable version
 -   Google Chrome™ most recent stable version
--   Microsoft® Edge® most recent stable version
 
 ### Supported browsers
 
+-   Chromium™ based browsers such as Microsoft® Edge® and Opera®, most recent stable version, desktop *and* tablet
```

Comment:
Edge is not yet based on Chromium, but it's coming later this year so this is just making it clear we won't waste time on deprecated versions of Edge using own layout engine.